external/wakaama/examples/shared/Make.defs

```diff
@@ -38,5 +38,9 @@
 CSRCS += commandline.c connection.c platform.c memtrace.c
 CFLAGS+=-I$(TOPDIR)/../external/wakaama/core
 
+ifeq ($(CONFIG_LWM2M_SECURITY),y)
```

Comment:
If i recall correctly, there used to be a tinyDTLS option for wakaama as well. If we include the tinyDTLS option as well for UDP, what will CONFIG_LWM2M_SECURITY indicate?